pycsw/core/repository.py

```diff
@@ -300,9 +300,7 @@ def insert(self, record, source, insert_date):
             self.session.commit()
         except Exception as err:
             self.session.rollback()
-            msg = 'Cannot commit to repository'
-            LOGGER.exception(msg)
-            raise RuntimeError(msg)
+            raise
```

Comment:
Will (only) `raise` emit `msg`?

Reply:
Yup, this just re-raises the same exception that was caught in the `except` block. This was used so I could catch DB errors in the `admin.load_records` function and return a specific error rather than a generic message/exception.